### PR TITLE
Add support.md file

### PR DESCRIPTION
**Pull request by dhensby**
I think it'd be good for us to add a specific file outlining lines of support.

see https://github.com/blog/2400-support-file-support

Content for review

### CODE REVIEW

**Comment by tractorcow** [comment]
Sounds good to me. Maybe @sminnee wants to review from a legalese point of view?

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+You can find support via:
+
+- [Stack Overflow (remember to tag as silverstripe)](http://stackoverflow.com/questions/tagged/silverstripe)
+- [Slack](http://silverstripe.org/slack)
+
+For more information please see our [community page](http://www.silverstripe.org/community/#introduction).
+
+## Commercial support
+
+If you require a higher level of support with an SLA, you may want to consider commercial support provided by SilverStripe Ltd.
```
Comment (chillu) [inline SUPPORT.md]: I think that's too one-sided - it should point to https://www.silverstripe.org/community/developer-and-partner-directory/.

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+If you require support there are two types of support available, community support and commercial support.
+
+## Community support
+
+Community support is provided by members of our open source community and is provided out of their good will. Please be kind, polite and corteous to those that try to help you.
+
+You can find support via:
+
+- [Stack Overflow (remember to tag as silverstripe)](http://stackoverflow.com/questions/tagged/silverstripe)
```
Comment (chillu) [inline SUPPORT.md]: Wanna add the forum?

**Comment by dhensby** [comment]
feedback implemented

**Pull request merged**